### DISCUSSION TITLE
Can Elsa support low-code forms?

### DISCUSSION THREAD

**Discussion by Jauhuei** [elsa-workflows/elsa-core#4335] [Q&A]
Front-end designers can drag and drop to create a form, and when the form is submitted, the corresponding workflow is triggered. 
Elsa itself should not have such support, but has anyone done a similar project? 
What third-party low-code front-end pages can work well with Elsa?

**Comment by sfmskywalker** [comment]
Yes, you can use Elsa workflows to handle form submissions. An easy approach would be to leverage the HTTP Endpoint activity to handle requests, but a more elegant approach would be to have an activity that represents a form submission, called e.g. `Form Submitted` that serves as both a workflow trigger to start new workflow instances and resume existing ones. There’s no guide for this at the moment, but I’m committed to creating one as soon as I can to serve as an example.

**Comment by mohdali** [comment]
Any library that can let you capture the form content in json format (r as a dictionary) would be fine.

Personally, I've done it with DevExpress Reports (paid) and using custom activity like @sfmskywalker explained
